src/Bridge/Doctrine/EventListener/PublishMercureUpdatesListener.php

```diff
@@ -190,7 +196,35 @@ private function publishUpdate($object, array $targets): void
             $data = $this->serializer->serialize($object, key($this->formats), $context);
         }
 
-        $update = new Update($iri, $data, $targets);
-        $this->messageBus ? $this->dispatch($update) : ($this->publisher)($update);
+        $updates = array_merge([new Update($iri, $data, $targets)], $this->getGraphQlSubscriptionUpdates($object, $targets, $type));
+
+        foreach ($updates as $update) {
+            $this->messageBus ? $this->dispatch($update) : ($this->publisher)($update);
+        }
+    }
+
+    /**
+     * @param object $object
+     *
+     * @return Update[]
+     */
+    private function getGraphQlSubscriptionUpdates($object, array $targets, string $type): array
+    {
+        if ('update' !== $type || !$this->graphQlSubscriptionManager || !$this->graphQlMercureSubscriptionUriGenerator) {
+            return [];
+        }
+
+        $payloads = $this->graphQlSubscriptionManager->getPushPayloads($object);
+
+        $updates = [];
+        foreach ($payloads as [$subscriptionId, $data]) {
+            $updates[] = new Update(
+                $this->graphQlMercureSubscriptionUriGenerator->generateMercureTopicUrl($subscriptionId),
+                $this->serializer->serialize($data, key($this->formats)),
```

Comment:
@alanpoulain Wouldn't this serialize the messages using the same format (defaults to JSON-LD I believe?) as the normal mercure updates? I think it would make sense to use GraphQL here, no?

Reply:
The normalized data is sent as JSON like `graphql-php` would do. Maybe I should use `json` instead of the key of formats, WDYT?

Reply:
I've changed it to mimic how the endpoint is doing it: https://github.com/api-platform/core/blob/9298c4502b56f3bff7ed91ad05ea71ca216b58cf/src/GraphQl/Action/EntrypointAction.php#L85.

Reply:
LGTM 👍